p2p/p2p.go

```diff
@@ -133,37 +133,17 @@ func (s *Service) Start() error {
 
 // start begins the p2p Service, including discovery. start does not terminate once called.
 func (s *Service) start(e chan error) {
-	if len(s.bootstrapNodes) == 0 && !s.noBootstrap {
+	if len(s.bootnodes) == 0 && !s.noBootstrap {
 		e <- errors.New("no peers to bootstrap to")
+		return
 	}
 
-	// this is in a go func that loops every minute due to the fact that we appear
-	// to get kicked off the network after a few minutes
-	// this will likely be resolved once we send messages back to the network
-	go func() {
```

Comment:
Is this no longer applicable?

Reply:
It's certainly a bandaid if it is. Lets rip that sucker off!